xc-admin/packages/xc-admin-common/src/multisig_transaction/WormholeMultisigInstruction.ts

```diff
@@ -47,12 +50,12 @@ export class WormholeMultisigInstruction implements MultisigInstruction {
 
       if (result.name === "postMessage") {
         try {
-          const decoded = decodeGovernancePayload(result.args.payload);
-          result.args.governanceName = decoded.name;
-          result.args.governanceArgs = decoded.args;
+          const decoded: PythGovernanceAction = decodeGovernancePayload(
```

Comment:
I'm gonna try to switch to a paradigm where it doesn't throw errors in a future PR